src/view_page.cpp

```diff
@@ -487,17 +559,28 @@ void View::DrawBracket(DeviceContext *dc, int x, int y1, int y2, int staffSize)
     assert(dc);
 
     int x1, x2;
-
-    x2 = x - m_doc->GetDrawingBeamWidth(100, false);
-    x1 = x2 - m_doc->GetDrawingBeamWidth(100, false);
+    if (m_doc->GetType() == Facs) {
+        x2 = x - m_doc->GetDrawingBeamWidth(staffSize, false);
```

Comment:
Where do you get the staffSize from. Cannot we use it in all cases?

Reply:
`staffSize` comes from from `last->m_drawingStaffSize` on line 353/414 in the same file.